Formula/bup.rb

```diff
@@ -36,16 +36,33 @@ class Bup < Formula
     sha256 "0fe2d45ba43b00a41cd73f8be321a44936dc1aba233dee979f17a042b83eb6dc"
   end
 
+  if OS.linux?
+    resource "pyxattr" do
+      url "https://files.pythonhosted.org/packages/cf/b1/7ed931d98b5a91a59b69fcc2860e5b720a22ed1ddb85268415181c9b0986/pyxattr-0.7.1.tar.gz"
+      sha256 "965388dd629334e850aa989a67d2360ec8257cfe8f67d07c29f980d3152f2882"
+    end
+  end
+
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
+    ENV.append "CFLAGS", " #{`python3 --cflags`.chomp}"
```

Comment:
This should be `python3-config` not `python3`.
Also, there is no need to add a space like you're doing:

```suggestion
    ENV.append "CFLAGS", `python3 --cflags`.chomp
```